### PR TITLE
Fixing issue #10 on bob.io.base

### PR DESCRIPTION
**Pull request by tiagofrepereira2012**
The original issue is here https://github.com/bioidiap/bob.io.base/issues/10

### CODE REVIEW

**Comment by 183amir** [comment]
Can you do it like this: https://github.com/conda-forge/bob.core-feedstock/tree/master/recipe

**Comment by 183amir** [comment]
Your patch seems to be working. Could you please rerender the feedstock too?
-     Clone your fork feedstock on your computer.
-     Install conda-smithy from the conda-forge channel into the root environment. `conda install -n root -c conda-forge conda-smithy`
-     Run `conda-smithy rerender`.
-     Add and comit.
-     Push to update the PR on the feedstock here.

If you have questions, please ask.

**Comment by 183amir** [comment]
@conda-forge/core what do you think about this pull? we managed to fix the OSX builds.

**Comment by pelson** [comment]
Thanks for the ping. Seems pretty sensible to me.

File: recipe/build.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+if [[ `uname` == 'Darwin' ]]; then
+	CXXFLAGS="${CXXFLAGS} -DBOOST_NO_CXX11_RVALUE_REFERENCES"
+	MACOSX_DEPLOYMENT_TARGET=10.9
+	$PYTHON -B setup.py install --single-version-externally-managed --record record.txt
```
Comment (pelson) [inline recipe/build.sh:6]: This could be pulled out of the if statement.

Comment (183amir) [inline recipe/build.sh:6]: @tiagofrepereira2012 could you please take `$PYTHON -B setup.py install --single-version-externally-managed --record record.txt` line out of the `if` statement.

File: recipe/build.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+if [[ `uname` == 'Darwin' ]]; then
+	CXXFLAGS="${CXXFLAGS} -DBOOST_NO_CXX11_RVALUE_REFERENCES"
```
Comment (jakirkham) [inline recipe/build.sh:4]: Do we still need this?

File: recipe/build.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+	MACOSX_DEPLOYMENT_TARGET=10.9
```
Comment (jakirkham) [inline recipe/build.sh:5]: Should be exported, no?

File: recipe/build.sh (hunk 1 above)
Comment (tiagofrepereira2012) [inline recipe/build.sh:6]: I guess you already did @183amir 
Sorry for the delay, I was in a rush today

Any issue, let me know

Comment (183amir) [inline recipe/build.sh:6]: No I didn't, I don't have access to your fork.

**Pull request closed without merging**